86BoxManager/dlgSettings.cs

```diff
@@ -18,26 +18,46 @@ public dlgSettings()
 
         private void dlgSettings_Load(object sender, EventArgs e)
         {
-            LoadSettings();
-            Get86BoxVersion();
+            ApplicationSettings.LoadSettings();
 
+            UpdateControlsToFitSettings();
+            Display86BoxVersion();
             lblVersion1.Text = Application.ProductVersion.Substring(0, Application.ProductVersion.Length - 2);
 
             #if DEBUG
                 lblVersion1.Text += " (Debug)";
             #endif
         }
 
+        /// <summary>
+        /// Update the controls to fit the new settings in the static Settings class.
+        /// </summary>
+        private void UpdateControlsToFitSettings()
```

Comment:
You're not disabling the logging textbox and browse button if logging is disabled.
Minimize on VM start checkbox somehow depends on the Minimize to tray icon setting/checkbox state.